lib/demos/material/text_field_demo.dart

```diff
@@ -106,6 +110,28 @@ class _PasswordFieldState extends State<PasswordField> {
 class TextFormFieldDemoState extends State<TextFormFieldDemo> {
   PersonData person = PersonData();
 
+  FocusNode _phoneNumber, _email, _lifeStory, _password, _retypePassword;
+
+  @override
+  void initState() {
+    _phoneNumber = FocusNode();
+    _email = FocusNode();
+    _lifeStory = FocusNode();
+    _password = FocusNode();
+    _retypePassword = FocusNode();
+    super.initState();
```

Comment:
Please call super first for `initState()`, it is fine to call last for `dispose()`